### ISSUE TITLE
Inconsistent yaml formatting

### ISSUE THREAD

**Issue by quasarea** [prettier/prettier-vscode#1932]
**VS Code Version**: 1.55.2
**Prettier Extension Version**: 6.3.2.
**OS and version**: W10 1903

### Summary

Given yaml like:
```yaml
  - stage: Process
    jobs:
      - template: Process.yml
        parameters:
          ? ${{ if in(parameters.BuildType, 'a') }}
          : BuildArtifacts:
              - input: Build.Release.x86
                output: Processed.Release.x86                   
          ? ${{ if in(parameters.BuildType, 'b                               ') }}
          : BuildArtifacts:
              - input: Build.Release
                output: Processed.Release
```

When formatting the first ?: gets reformatted while the second is left as is. It seems to depend on line length (happens only for shorter than 84 characters) but I have not found any place to configure/control this behavior. Due to that I'm ending up with inconsistent code like:

```yaml
  - stage: Process
    jobs:
      - template: Process.yml
        parameters:
          ${{ if in(parameters.BuildType, 'a') }}:
            BuildArtifacts:
              - input: Build.Release.x86
                output: Processed.Release.x86
          ? ${{ if in(parameters.BuildType, 'b                               ') }}
          : BuildArtifacts:
              - input: Build.Release
                output: Processed.Release
```
for above example.

**Comment by ntotten** [comment]
This is an issue or feature request for the Prettier library itself rather than the VS Code extension for prettier. Please open an issue at https://github.com/prettier/prettier.

**Comment by quasarea** [comment]
thanks, submitted as https://github.com/prettier/prettier/issues/10822

**Comment by quasarea** [comment]
@ntotten new prettier with fix was released as v2.3.1
